opacus/grad_sample/conv.py

```diff
@@ -20,60 +21,41 @@ def compute_conv_grad_sample(
     """
     Computes per sample gradients for convolutional layers
 
-    Args:
+    args:
```

Comment:
According to Ashkan's docstring style guide we have to use capital `Args`. I will add those guidelines in the docs.